src/command/output.rs

```diff
@@ -36,6 +41,37 @@ impl RoverStdout {
                 }
                 println!("{}", &hash);
             }
+            RoverStdout::SubgraphList(details) => {
+                if atty::is(Stream::Stdout) {
+                    tracing::info!("Subgraphs:");
```

Comment:
Should this be `tracing::info`, or can it be just `println`? I am mainly asking because it seems like the actual list will not be added to the trace, which maybe confusing from a logging point of view.

Also the tiniest of nits: how do you feel about having a `\n\n` after subgraph for some 'breathing room' in display :D :D